cnd/src/start_swap.rs

```diff
@@ -246,12 +221,109 @@ where
                     },
                 })
                 .await?;
+        }
+        _ => tracing::info!("attempting to save for an unsupported swap"),
+    };
+
+    start_swap(&spawner, &storage, swap).await?;
```

Comment:
We shouldn't start the swap if we hit this catch-all I reckon.

Reply:
Better to bail from this function :)

Reply:
Does this still apply to https://github.com/comit-network/comit-rs/pull/2796 ? (brain not able to think any more)

Reply:
https://github.com/comit-network/comit-rs/pull/2796